nav_msgs/OccupancyGrid grid
---
std_msgs/String str